src/App/components/Login/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+import { login } from '../../../services/userService';
+import LoginLayout from './layout';
+import history from '../../../history';
+import { setLoginInfo } from '../../../auth';
+
+class Login extends Component {
+  state = {
+    showError: false
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const user = {
+      email: event.target.user.value,
+      password: event.target.password.value
+    };
+    login(user).then((response) => {
+      setLoginInfo(response.data);
+      history.push('/dashboard');
```

Comment:
Why don't you use react-router to change the location?

Reply:
This is the way you change the route with React Router 4 :disappointed: